sdk/deepbook-v3/src/utils/constants.ts

```diff
@@ -80,30 +80,30 @@ export const mainnetCoins: CoinMap = {
 export const testnetPools: PoolMap = {
 	DEEP_SUI: {
 		address: `0x67800bae6808206915c7f09203a00031ce9ce8550008862dda3083191e3954ca`,
-		baseCoin: 'DEEP',
-		quoteCoin: 'SUI',
+		baseCoin: testnetCoins.DEEP,
+		quoteCoin: testnetCoins.SUI,
```

Comment:
this makes sense again now.  I had removed these, because the coins were previously mutable with IDs, but I pulled the IDs out so I think this makes more sense.